libparsec/crates/platform_storage/tests/unit/workspace.rs

```diff
@@ -1081,9 +1081,9 @@ async fn set_prevent_sync_pattern(env: &TestbedEnv) {
     let res = workspace.set_prevent_sync_pattern(&regex).await.unwrap();
 
     assert_eq!(res, false);
-    let (got_regex, bool) = workspace.get_prevent_sync_pattern().await.unwrap();
+    let (got_regex, bool) = workspace.get_prevent_sync_pattern();
 
-    assert_eq!(got_regex, regex);
+    assert_eq!(got_regex, &regex);
     assert_eq!(bool, false);
```

Comment:
Should also restart the workspace to ensure the prevent sync pattern is not only in cache

```rust
// Restart the workspace to ensure the new prevent sync pattern is not just in cache
workspace.stop().await.unwrap();
let workspace = start_workspace(env).await;
let (got_regex, bool) = workspace.get_prevent_sync_pattern();
assert_eq!(got_regex, &regex);
assert_eq!(bool, false);
```

Reply:
done